include/mbedtls/mbedtls_config.h

```diff
@@ -1141,7 +1141,10 @@
  *
  * Enable support for PKCS#1 v2.1 encoding.
  *
- * Requires: MBEDTLS_MD_C, MBEDTLS_RSA_C
+ * Requires: MBEDTLS_RSA_C and (MBEDTLS_MD_C or MBEDTLS_PSA_CRYPTO_C).
```

Comment:
Should this be
```suggestion
 * Requires: MBEDTLS_RSA_C and (MBEDTLS_MD_C or ( MBEDTLS_PSA_CRYPTO_C and PSA_WANT_ALG_MD5) ).
```
?

Reply:
Now, there's nothing specific about MD5 here. However, I agree that we should clarify that if using MD, you also need the `MBEDTLS_xxx_C` for all the hashes you can to use, and otherwise you also need `PSA_WANT_ALG_xxx` for those hashes.